static/app/views/alerts/rules/metric/ruleForm.tsx

```diff
@@ -998,6 +1010,10 @@ class RuleFormContainer extends DeprecatedAsyncComponent<Props, State> {
 
     const showMigrationWarning =
```

Comment:
Let's also rename it here?
```suggestion
    const showTransactionMigrationWarning =
```